visualizer/index.html

```diff
@@ -107,19 +107,18 @@
     stroke: #D0D0D0;
   }
 
-  .timeline.unrefed .wait,
-  .timeline.unrefed .callback {
-    opacity: 0.5;
-  }
-
   .timeline .init {
     stroke-width: 2px;
     stroke: rgba(0, 0, 0, 0.7);
   }
-  .timeline .wait {
+  .timeline .wait,
```

Comment:
I don't think we need two `.wait` classes, we just need an `.unref` and `.ref` class. Such that `.wait.unref` and `.wait.ref` can be used for selection.